add-ons/grafana/grafana.addon

```diff
@@ -4,7 +4,7 @@
 # Required-Vars: namespace 
 
 # TODO: Once conditional commands are there, create the namespace if it does not exist
-oc adm new-project #{namespace}
+#oc adm new-project #{namespace}
```

Comment:
`#` comment out this step but if you want to run it but ignore the failure then better to use `!`

Reply:
@cunningt Also now `oc new-project` works without any issue.